apis/nucleus/src/components/Header.jsx

```diff
@@ -67,20 +57,12 @@ function Header({ layout, sn, anchorEl, hovering, focusHandler, titleStyles = {}
     });
   }, [sn]);
 
-  useEffect(() => {
-    if (!containerRect) return;
-    const { width } = containerRect;
-    setShouldShowPopoverToolbar(width < MIN_WIDTH);
-  }, [containerRect]);
-
   const showTitles = showTitle || showSubtitle;
   const cls = [classes.containerStyle, ...(showTitles ? [classes.containerTitleStyle] : [])];
-  const showPopoverToolbar = (hovering || showInSelectionActions) && (shouldShowPopoverToolbar || !showTitles);
-  const showToolbar = showTitles && !showPopoverToolbar && !shouldShowPopoverToolbar;
+  const showPopoverToolbar = hovering || showInSelectionActions;
 
   const Toolbar = (
     <ActionsToolbar
-      show={showToolbar}
```

Comment:
maybe a `show={false}` (default is true)
in case there are any object that tries to show actions when not in selection mode

Reply:
Yea, ok. Not sure what scenario will be like that, but I will add it